stubs/3.2/queue.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any
 
 class Queue():
+    @staticmethod
```

Comment:
Since you merged it, I guess you figured out why I did this. (Type of the first parameter was being changed to `Queue`, because the method was neither static or a class method.)